agent/agent.go

```diff
@@ -4318,6 +4327,14 @@ func (a *Agent) registerCache() {
 		RefreshTimeout: 10 * time.Minute,
 	})
 
+	a.cache.RegisterType(cachetype.ConfigEntryName, &cachetype.ConfigEntry{
```

Comment:
We are not actually using this cache type anymore, could just leave it in the codebase for future use or remove it.

Opinions wanted!

Reply:
Removed it, bye bye!